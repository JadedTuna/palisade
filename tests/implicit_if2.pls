in {
    high x: bool;
}
out {
    low y: int;
}

if (x) {
    while (true) {
        y = y + 1;
    }
}
