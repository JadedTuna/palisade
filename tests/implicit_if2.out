[1;35msecurity error: [0minsecure implicit flow - while loop with a high guard
   7 | 
   8 | if (x) {
   9 |     while ([1;35mtrue[0m) {
       [1;35m~~~~~~~~~~~[0m[1;35m^^^^[0m
